src/tasks/stats.js

```diff
@@ -1,5 +1,5 @@
 const { Task, config } = require("../index");
-const snekfetch = require("snekfetch");
+const fetch = require("node-fetch");
```

Comment:
should be using fetch util if this was the case but its a Piece so use this.fetchURL